index.html

```diff
@@ -755,9 +755,11 @@ <h4>Transformation (ecdsa-rdfc-2019)</h4>
 Let |canonicalDocument| be the result of converting |unsecuredDocument| to
 <a data-cite="JSON-LD11-API#expansion-algorithm">JSON-LD expanded form</a>
 and then <a data-cite="JSON-LD11-API#deserialize-json-ld-to-rdf-algorithm">
-to RDF statements</a>, applying the <a data-cite="RDF-CANON#canon-algorithm">
-RDF Dataset Canonicalization Algorithm</a>, and then serializing the
-result to [[N-QUADS]].
+to RDF statements</a>, applying the <a data-cite="RDF-CANON#canon-algorithm">RDF Dataset Canonicalization
+Algorithm</a> [[RDF-CANON]] to the result, and then serializing the result to a
+<a data-cite="RDF-CANON#dfn-serialized-canonical-form">serialized canonical form</a> [[RDF-CANON]].
+For canonicalization one must use the hash algorithm appropriate in security level to
```

Comment:
This "must" would be a new normative statement and would need to be capitalized ... perhaps this instead?

```suggestion
For canonicalization one is expected use the hash algorithm appropriate in security level to
```

Reply:
This includes @dlongley's suggestion. I think an indefinite article (`a hash`) is appropriate here, but I might be convinced that a definite article (`the hash`) is better. Such convincing would start with a full edit of this sentence to incorporate that definite article
```suggestion
For canonicalization, one is expected to use a hash algorithm that has an appropriate security level for
```